opslib/osm/charm.py

```diff
@@ -64,15 +64,24 @@
     env |
     grep OSM |
     sed -e 's/^/export /' |
-    sed -e 's/$$/\\\"/' |
-    sed -e 's/=/=\\\"/' |
+    sed -e 's/$$/\"/' |
+    sed -e 's/=/=\"/' |
     tee -a /root/.bashrc
 )
+echo "[[ \`which code\` &&  
```

Comment:
no need to escape `

Reply:
I do need to escape it. If you execute `echo "[[ `which code`... ` it will evaluate the `which code` command.